packages/pluggableWidgets/activity-indicator-native/src/ActivityIndicator.editorConfig.ts

```diff
@@ -0,0 +1,19 @@
+import { RowLayoutProps } from "../../../tools/piw-utils/src/typings";
+import { ActivityIndicatorPrimarySVG } from "./assets/ActivityIndicatorPrimary";
+
+export const getPreview = (): RowLayoutProps => ({
```

Comment:
We are basically exporting a function here. I think it's better to do `export function getPreview...`